client/app/components/ArticleList.jsx

```diff
@@ -4,17 +4,22 @@ import Article from './Article.jsx';
 const ArticleList = ({ storyPoint }) => {
   let articles;
 
-  !storyPoint.hasOwnProperty('stories')
-  ? articles = <div></div>
-  : articles = (
+  if (storyPoint.hasOwnProperty('stories') && storyPoint.stories[0] === null) {
+    articles = <div>
```

Comment:
style suggestion to use parens block for multi-line html assignment like on line 12